plugins/modules/postgresql_pg_hba.py

```diff
@@ -90,13 +90,38 @@
     type: str
     default: sdu
     choices: [ sdu, sud, dsu, dus, usd, uds ]
+  overwrite:
+    description:
+      - "Remove all existing rules before adding rules. (Like C(state: absent) for all pre-existing rules.)"
+    type: bool
+    default: false
   keep_comments_at_rules:
     description:
       - If C(true), comments that stand together with a rule in one line are kept behind that line.
       - If C(false), such comments are moved to the beginning of the file, like all other comments.
     type: bool
     default: false
     version_added: '1.5.0'
+  rules:
+    description:
+      - >-
+        A list of objects, specifying rules for the pg_hba.conf. Use this to manage multiple rules at once.
+        Each object can have the following keys (the "rule-specific arguments"), which are treated the same as if they
+        were arguments of this module:
```

Comment:
```suggestion
      - A list of objects, specifying rules for the pg_hba.conf. Use this to manage multiple rules at once.
      - "Each object can have the following keys (the 'rule-specific arguments'), which are treated the same as if they were arguments of this module:"
```
For better readability